api/PictureInPictureEvent.json

```diff
@@ -0,0 +1,218 @@
+          "opera": {
+            "version_added": "67"
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": "13.1"
+          },
+          "safari_ios": {
+            "version_added": "14"
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": "72"
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "PictureInPictureEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PictureInPictureEvent/PictureInPictureEvent",
+          "description": "<code>PictureInPictureEvent()</code> constructor",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "69",
+                "version_removed": "72",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "enable-experimental-web-platform-features",
+                    "value_to_set": "enabled"
+                  },
+                  {
+                    "type": "preference",
+                    "name": "enable-surfaces-for-videos",
+                    "value_to_set": "enabled"
+                  },
+                  {
+                    "type": "preference",
+                    "name": "enable-picture-in-picture",
+                    "value_to_set": "enabled"
+                  }
+                ]
+              },
+              {
+                "version_added": "72"
+              }
+            ],
+            "chrome_android": {
+              "version_added": "72"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "67"
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": "13.1"
+            },
+            "safari_ios": {
+              "version_added": "14"
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": "72"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "relatedTarget": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PictureInPictureEvent/relatedTarget",
```

Comment:
Maybe this should be `pictureInPictureWindow`?
https://w3c.github.io/picture-in-picture/#event-types

Reply:
That's what it's called in Chrome. If it's still `relatedTarget` on some platforms you'll need to record an `alternative_name`.

Reply:
This was definitely a copy paste issue taken from another file. I do not believe that `alternative_name` would be required here 

That's been changed now